projects/memapGui/src/main/java/fortiss/simulation/Parameters.java

```diff
@@ -212,6 +212,8 @@ public void setData() {
 				this.priceData = new TimedDataAdapter(new TimedData(fm.readFromResources(ef.getFile(location)), labels));
 			} else {
 				this.priceData = new TimedDataAdapter(new TimedData(fm.readFromSource(location), labels));
+				System.out.println(Parameters.class + " new data loaded.");
+				System.out.println(Parameters.class + " data: " + this.priceData.getYValues("Price [EUR/kWh]"));
```

Comment:
Are these comments intended for debugging? If so, they can probably be removed. If not, then maybe it would be better to use the lof feature.

Reply:
yes, i remove them.